datacontract/export/spark_converter.py

```diff
@@ -128,7 +128,9 @@ def to_data_type(field: Field) -> types.DataType:
     if field_type in ["string", "varchar", "text"]:
         return types.StringType()
     if field_type in ["number", "decimal", "numeric"]:
-        return types.DecimalType(precision=field.precision, scale=field.scale)
+        precision = field.precision if field.precision is not None else 10
```

Comment:
default is 38.

Reference: https://datacontract.com/#field-object

Reply:
Thanks for the input, @jochenchrist! Changed the default value for `precision` to `38` now.